<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>路径设置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/loadtest.png</normaloff>images/loadtest.png</iconset>
  </property>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOut">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择Excel输出路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOut">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\处理后的文件</string>
   </property>
   <property name="placeholderText">
    <string>输出Excel路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInput">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\原文件</string>
   </property>
   <property name="placeholderText">
    <string>输入Xml路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonInput">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择Xml输入路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOut_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择Xml输出路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOut_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\处理后的文件</string>
   </property>
   <property name="placeholderText">
    <string>输出Xml路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOut_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择MPR输出路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOut_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\处理后的文件</string>
   </property>
   <property name="placeholderText">
    <string>输出MPR路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonInput_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择MPR输入路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInput_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\原文件</string>
   </property>
   <property name="placeholderText">
    <string>输入MPR路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonOut_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择CSV存放路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOut_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>D:\处理后的文件</string>
   </property>
   <property name="placeholderText">
    <string>选择csv存放路径</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>浮动铣刀</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
